spark/src/test/resources/mushrooms.arff

```diff
@@ -0,0 +1,8301 @@
+% 1. Title: Mushroom Database
```

Comment:
All the test data are in `shell/src/universal/data` directory.

Reply:
done